DEVELOPMENT.md

```diff
@@ -316,6 +322,61 @@ When editing or creating usage documentation, add headlines and appropriate cont
 the table of contents.
 
 
+## Code Style
+
+Most code style is enforced through ESLint, TSLint and Stylelint, so make sure to run all of the linters.
+
+### Lint configurations
+
+- [ESLint configuration][eslint-config]
+- [TSLint configuration][tslint-config]
+- [Stylelint configuration][stylelint-config]
+
+### Member Ordering
+
+When writing a class, this is the basic order of items:
+
+1. Private properties
+1. Public properties
+1. Getters (getter goes before setter, which is before the associated private property)
+1. View references (`@ViewChild` etc)
+1. `@Input`s
+1. `@Output`s
+1. Constructor
+1. Lifecycle methods (in the order they are called by Angular) (`ngOnInit` etc)
+1. Public methods
+1. Protected or static methods
+1. Private methods
+
+### Underscores
+
+Prefixing private methods and properties is a common pattern. We have opted to rely on TypeScript's member access rather than the
+ underscore method.
+
+> Caveat: The one place we do use the underscore is to prefix the private property that is used with a getter and setter:
+
+```typescript
+public set active(v: boolean) {
+  this._foo = v;
+}
+public get active(): boolean {
+  return this._foo;
+}
+private _foo = false;
+```
```

Comment:
But you just said getter goes before setter 😂 
I think our code base has setter first, then getter..

Reply:
Whoops. Good catch.

Reply:
Updated